app.rb

```diff
@@ -0,0 +1,181 @@
+require 'colorize'
+require_relative 'student'
```

Comment:
- [ ] When I run the app, I get the below error. If I comment the line `require 'colorize'` the app works fine. Please fix this issue.

![image](https://user-images.githubusercontent.com/83803414/199728065-2b4dd7d4-1d56-4afa-9428-345998c10c39.png)